data/json/itemgroups/vending_machines.json

```
@@ -130,5 +130,303 @@
     "subtype": "collection",
     "id": "vending_medicine",
     "entries": [ { "group": "drugs_vending", "count-min": 10, "count-max": 20 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_adult",
+    "items": [
+      { "item": "condom", "prob": 30, "count": [ 1, 5 ] },
+      { "group": "ss_merch_1", "prob": 20 },
+      { "group": "ss_merch_2", "prob": 20 },
+      { "group": "ss_merch_3", "prob": 20 },
```

`ss_merch_3` will drop entire fursuits inside a vending machine, while `ss_merch_2` is redunant as it only contains porn mags. Odd that group doesn't at least have a chance of spawning erotic novels or some other variety to its contents. XD
```suggestion
      { "group": "novel_erotic", "prob": 20, "count": [ 1, 2 ] },
```